bench-run/publication/arg_parser.py

```diff
@@ -0,0 +1,10 @@
+import argparse
```

Comment:
1. Why not just `publish` instead of `publication`?
2. Looks like this module is not used anywhere.

Reply:
fixed